currency.go

```diff
@@ -0,0 +1,81 @@
+package currency
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// ConvertValueFromString : returns a uint from a string value
+func ConvertValueFromString(num string) (uint, error) {
+	reg := regexp.MustCompile("[0-9]+")
+	str := reg.FindAllString(num, -1)
+	newStr := strings.Join(str, "")
+	output, err := strconv.ParseUint(newStr, 10, 64)
+	if err != nil {
+		return 0, errorUnableToFormatCurrencyFromString
+	}
+	return uint(output), nil
+}
+
+// ConvertToStringFull : returns currency string with full formatting
+func ConvertToStringFull(num uint, alpha string) (string, error) {
+	ISO, err := getISOFromAlpha(alpha)
+	if err != nil {
+		return "", err
+	}
+	output := formatCurrency(num, ISO)
+	return output, nil
+}
+
+// ConvertToStringAlpha : returns currency string formatting with ISO alpha code instead of symbol
+func ConvertToStringAlpha(num uint, alpha string) (string, error) {
```

Comment:
Even if we don't end up using it, i like the idea of keeping it in just because it's not any additional work and someone else might find it useful at some point. 

[https://www.freeformatter.com/united-states-standards-code-snippets.html](url)

Apparently its an Alpha-3 value. What do you think about "DisplayWithAlpha"or "DisplayWithAlpha-3" something like that?